libs/mocks/WordPressMocks/src/main/assets/mocks/mappings/wpcom/stats/stats_top-posts.json

```diff
@@ -6,17 +6,17 @@
     "response": {
         "status": 200,
         "jsonBody": {
-            "date": "{{now format='yyyy-MM-dd'}}",
+            "date": "{{fnow format='yyyy-MM-dd'}}",
             "days": {
-                "{{now format='yyyy-MM-dd'}}": {
+                "{{fnow format='yyyy-MM-dd'}}": {
                     "postviews": [
                         {
                             "id": 0,
                             "href": "https://e2eflowtestingmobile.wordpress.com/",
                             "date": null,
                             "title": "Home page / Archives",
                             "type": "homepage",
-                            "views": 183,
+                            "views": 1832,
```

Comment:
Why this change?

Reply:
Whoops. This is a leftover from my debugging session, when I was trying to ensure that WireMock indeed used the `.json` file I thought it was using. Should be reverted indeed, good catch.